docs/api/rest_api_authentication.md

```diff
@@ -0,0 +1,74 @@
+# REST API reference authentication basics
+
+This page refers to [REST API reference](https://doc.ezplatform.com/rest-api-reference), where you can find detailed information about
+REST API resources and endpoints.
+
+Use of HTTPS for authenticated (REST) traffic is highly recommended.
+
+## Basic Authentication
+
+For more information, see [HTTP Authentication: Basic and Digest Access Authentication.](http://tools.ietf.org/html/rfc2617)
+
+## OAuth
+
+For more information, see [OAuth 2.0 protocol for authorization.](https://oauth.net/2/)
+
+## Session based Authentication
+
+The sessions are only created to re-authenticate the user (and perform authorization), not to hold session state in the service.
+Because of that, we consider this method supports AJAX based applications even if it violates the principles of RESTful services.
```

Comment:
```suggestion
Because of that, we regard this method as supporting AJAX-based applications even if it violates the principles of RESTful services.
```
?